packages/api/es/collections.js

```diff
@@ -64,23 +82,40 @@ class Collection extends BaseSearch {
         total: 0,
       };
 
-      // Can't aggregate on the _id but can on the _uid which is collection#_id
-      const esUid = `collection#${constructCollectionId(updatedRecord.name, updatedRecord.version)}`;
-
-      const matchingBucket = aggs.aggregations.hashes.buckets
-        .find((bucket) => bucket.key === esUid);
-
-      if (matchingBucket) {
-        updatedRecord.stats.total = matchingBucket.stats.doc_count;
-        matchingBucket.stats.count.buckets.forEach((s) => {
+      const aggregations = get(aggs, 'aggregations.collections');
+      const nameBucket = this._filterAggBuckets(
+        aggregations.name.buckets, (agg) => agg.key === updatedRecord.name
+      );
+      const versionBucket = this._filterAggBuckets(
+        get(nameBucket, 'version.buckets'), (agg) => agg.key === updatedRecord.version
+      );
+
+      if (versionBucket) {
+        const stats = versionBucket.stats;
+        updatedRecord.stats.total = stats.doc_count;
+        stats.count.buckets.forEach((s) => {
           updatedRecord.stats[s.key] = s.doc_count;
         });
       }
-
       return updatedRecord;
     });
   }
 
+  _filterAggBuckets(buckets, filter) {
```

Comment:
I thought about using #private from 2019, but that's not strictly required here.   Also typescript migration.